doc/source/_templates/autosummary/class.rst

```diff
@@ -18,12 +18,12 @@
       .. autosummary::
          :toctree:
       {% for item in all_methods %}
-         {%- if not item.startswith('_') or item in ['__call__', '__mul__', '__getitem__', '__len__', '__pow__'] %}
+         {%- if not item.startswith('_') or item in ['__call__', '__mul__', '__getitem__', '__len__', '__pow__', '__matmul__', '__truediv__', '__add__', '__rmul__', '__rmatmul__'] %}
```

Comment:
Looks like this is showing up [here](https://output.circle-artifacts.com/output/job/153f2743-a569-444e-b814-05315d0edff5/artifacts/0/html/reference/generated/scipy.sparse.linalg.LinearOperator.html).

<img width="708" height="707" alt="image" src="https://github.com/user-attachments/assets/3041f1ff-bf42-471d-848d-62a3d5991b67" />

Anywhere else I should check?